news/_posts/releases/2020-11-18-prestashop-1-7-7-0-available.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "PrestaShop 1.7.7.0 is available"
+subtitle: "You waited for it, here it is!"
+date:   2020-11-18 16:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/11/build_and_mail_final.jpg
+icon: icon-leaf
+tags:
+ - version
+ - 1.7
+ - 1.7.7.x
+ - minor
+ - releases
+---
+
+We are happy to announce PrestaShop 1.7.7.0 is officially available!
+
+![1.7.7.0 is available!](/assets/images/2020/11/build_and_mail_final.jpg)
+
+This release is our biggest yet since 1.7.1.0. More than one year of development, two betas, one release candidate Was it worth it? We absolutely think so, and hope that you will enjoy using it.
+
+## News in the 1.7.7.0
+
+### Productivity
+
+The **[full redesign and the new features of order pages]**(https://build.prestashop.com/news/the-new-order-pages-in-prestaShop-1-7-7/) help merchants to make a better use of their time and to stay focus on their online business. 
+
+The redesign of the user interface allows merchants to find the right information quickly while the new features allow a better efficiency on daily tasks and avoid back and forth between pages. 
+
+### Growth and flexibility
+
+PrestaShop is an international solution and we aim at providing highly global features for worldwide users. 
+
+The **[international improvements]**(https://build.prestashop.com/news/prestashop-keeps-growing-internationally/) remove barriers to go international and help merchants reach more customers.
+
+**[The fuzzy search]**(https://build.prestashop.com/news/introduction-to-the-fuzzy-search/) improve both the user experience and the conversion rate with a new search algorithm that takes misspellings or error inputs into account.
+
+In order to allow merchants to have all the currencies they need to meet the expectations of every customer, they can now **add new official and non official currencies** (e.g. local or custom) to their store and customize their display per language as desired.
```

Comment:
```suggestion
In order to allow merchants to have all the currencies they need to meet the expectations and trust of every customer, they can now **add new official and non-official currencies** (e.g. local or custom) to their store and customize their display per language as desired.
```

Reply:
```suggestion
In order to allow merchants to have all the currencies they need to meet the expectations and needs of every customer, they can now **add new official and non-official currencies** (e.g. local or custom) to their store and customize their display per language as desired.
```
Seems weird to talk of trust while it's a basic need for the customer to see the price in its own currency, it is more the elements of reinsurance or contact that will build confidence
Wdyt @MatShir ?